Makefile

```diff
@@ -1,14 +1,14 @@
 # Makefile for Planttracer web application
 # - Creates CI/CD environment in GitHub
 # - Manages deployemnt to AWS Linux
-# - Updated to handle virtual environment
-# - Simple CRUD management of local database instance for developers
```

Comment:
Did you intend to delete this comment?

It looks like some merge you did didn't Do The Right Thing.